ush/global_ens/evs_global_ens_atmos_grid2grid.sh

```diff
@@ -242,10 +259,21 @@ if [ $verify = upper ] ; then
                 echo "done" >> run_${modnam}_valid_at_t${vhour}z_${fhr}_${metplus_job}_g2g.sh
             fi
         fi
-        chmod +x run_${modnam}_valid_at_t${vhour}z_${fhr}_${metplus_job}_g2g.sh
+
+	# Indicate sub-task is completed for restart 
+	echo ">$COMOUTsmall/run_${modnam}_valid_at_t${vhour}z_${fhr}_${metplus_job}_g2g.completed" >> run_${modnam}_valid_at_t${vhour}z_${fhr}_${metplus_job}_g2g.sh
+
+	# Save files for restart
+        echo "mkdir -p $COMOUTsmall/restart/grid2grid/run_${modnam}_valid_at_t${vhour}z_${fhr}/stat" >> run_${modnam}_valid_at_t${vhour}z_${fhr}_${metplus_job}_g2g.sh
+	echo "cp -rfu $WORK/grid2grid/run_${modnam}_valid_at_t${vhour}z_${fhr}/stat/${modnam} $COMOUTsmall/restart/grid2grid/run_${modnam}_valid_at_t${vhour}z_${fhr}/stat" >> run_${modnam}_valid_at_t${vhour}z_${fhr}_${metplus_job}_g2g.sh
```

Comment:
Can we add two checks here?

1) SENDCOM = YES
2) The files it is trying to copy exist?

Reply:
1. `SENDCOM` is defaulted to YES in the [j-job](https://github.com/NOAA-EMC/EVS/blob/32258b087c1022efad02236c101b2bd022a2b96e/jobs/JEVS_GLOBAL_ENS_STATS#L14). Do you want to add `export SENDCOM = YES` in the driver scripts and ecf scripts?
2. Yes, the grid2grid jobs run the GenEnsProd, EnsembleStat, and GridStat sub-tasks in sequence and store the output netcdf and stat files in the `run_${modnam}_valid_at_t${vhour}z_${fhr}/stat/${modnam}` directory. So, the netcdf files from GenEnsProd already exist in the restart directory after the EnsembleStat and GridStat tasks are completetd.

Reply:
1. Before files are sent to COMOUT there should be checks that SENDCOM = YES (`if [ $SENDCOM = YES ]; then [copy files] fi)
2. We need to make sure that the files we want to copy exist in the source location. If they don't the copy command will fail and could cause a failure in operations.